lua/neo-tree/utils.lua

```diff
@@ -347,13 +347,6 @@ M.resolve_config_option = function(state, config_option, default_value)
   end
 end
 
----The file system path separator for the current platform.
```

Comment:
This part should not be removed, it is used in many places.

Reply:
it's duplicated, see
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/main/lua/neo-tree/utils.lua#L350
and
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/main/lua/neo-tree/utils.lua#L384

Reply:
OH! Good catch!